packages/cojson/src/localNode.ts

```diff
@@ -133,6 +133,59 @@ export class LocalNode {
     );
   }
 
+  static internalCreateAccount(opts: {
+    crypto: CryptoProvider;
+    initialAgentSecret?: AgentSecret;
+    peersToLoadFrom?: Peer[];
+  }): RawAccount {
+    const {
+      crypto,
+      initialAgentSecret = crypto.newRandomAgentSecret(),
+      peersToLoadFrom = [],
+    } = opts;
+    const accountHeader = accountHeaderForInitialAgentSecret(
+      initialAgentSecret,
+      crypto,
+    );
+    const accountID = idforHeader(accountHeader, crypto);
+
+    const node = new LocalNode(
+      initialAgentSecret,
+      crypto.newRandomSessionID(accountID as RawAccountID),
+      crypto,
+    );
+
+    for (const peer of peersToLoadFrom) {
+      node.syncManager.addPeer(peer);
+    }
```

Comment:
Moved the addPeer before the coValue creation in order to sync the coValues with just a content message.

If we add the peer after the flow would be "LOAD -> KNOWN empty -> CONTENT" because the sync would fall into the peer reconciliation flow.